.github/workflows/mdbook-build.yml

```diff
@@ -0,0 +1,37 @@
+name: MdBook Build and Deploy
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
`pull_request` shouldn't be here right? You only want to deploy this on a merge.

Reply:
If someone makes a PR for docs, it will be easy for you to go on checking live site.

Reply:
Where exactly would the mdbook be deployed in that case? Wouldn't that go straight into mfontanini.github.io/presenterm?

Reply:
It should be their own fork. But they need to setup gh pages for that. 

but nvm, you can remove it.